apps/vanilla/src/pages/2/src/views/view.ts

```diff
@@ -0,0 +1,26 @@
+import { createElement } from '../utils';
+
+/** View. */
+export abstract class View {
+	/** Returns template of view. */
+	public abstract get template(): string;
+
+	/** HTML element. */
+	protected abstract element: HTMLElement;
+
+	/** Returns HTML element. */
+	public getElement(): HTMLElement {
+		return this.element;
+	}
+
+	/**
+	 * Generates HTML element based on template.
+	 * @param template New template for view.
+	 */
+	protected generateElement(template?: string): HTMLElement {
+		if (template) {
```

Comment:
Try to avoid type casting in any view.

https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices#dont-use-implicit-type-casting

```suggestion
		if (template != null) {
```